ui/nuxeo-document-permissions/nuxeo-popup-permission.js

```diff
@@ -407,8 +407,8 @@ import '../widgets/nuxeo-user-suggestion.js';
       }
     }
 
-    _getResetParams() {
-      return {
+    _getParams() {
+      const resetedParams = {
```

Comment:
nit: I think `reseted` does not exists formally. I would suggest to rename it to `resetParams` (or even `baseParams`)